### PR TITLE
Create proc_creation_macos_suspicious_applet_behaviour.yml

### PR DESCRIPTION
**Pull request by D4rkCiph3r**
### Summary of the Pull Request
New rule for suspicious applet/OSA behaviour

### Detailed Description of the Pull Request / Additional Comments
This rule detects suspicious applet or command-line OSA behaviour probably dropped by malware into legitimate app's "/Contents/MacOS/" directory.
Refer: https://redcanary.com/blog/mac-application-bundles/ (XCSSET)

### Example Log Event
NA

### Fixed Issues
NA

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+tags:
+    - attack.t1059.002
+    - attack.execution
+logsource:
+    category: process_creation
+    product: macos
+detection:
+    selection:
+        ParentImage|endswith:
+            - '/applet '
```
Comment (frack113) [inline rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml:18]:
```suggestion
            - '/applet'
```

File: rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            - '/osascript '
```
Comment (frack113) [inline rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml:19]:
```suggestion
            - '/osascript'
```

**Review by frack113 — CHANGES_REQUESTED** [review]
I just remove the space

**Review by frack113 — APPROVED** [review]

File: rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+title: Suspicious Applet Behaviour
```
Comment (frack113) [inline rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml:1]:
```suggestion
title: Suspicious osacompile Run by Applet or Osascript
```

File: rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+id: a753a6af-3126-426d-8bd0-26ebbcb92254
+status: experimental
+description: Detects suspicious applet or command-line OSA behaviour probably dropped by malware into legitimate apps /Contents/MacOS/ directory
```
Comment (frack113) [inline rules/macos/process_creation/proc_creation_macos_suspicious_applet_behaviour.yml:4]:
```suggestion
description: | 
    Detects suspicious applet or osascript started a command-line with osacompile.
    Malware can use legitimate apps /Contents/MacOS/ directory to drop the payload
```

**Review by nasbench — APPROVED** [review]

**Pull request merged**